pages/account/security/security-specifications-dedicated-servers/guide.fr-fr.md

```diff
@@ -0,0 +1,217 @@
+
+Ce sont des sauvegardes de configurations des infrastructures qui permettent de délivrer le service aux clients telles que : configuration des routeurs, configurations vRack, affectation des IP, etc ...
+
+### 4.2 - Sauvegardes métier
+
+Liste des fonctionnalités et options de backups adaptées au service :
+
+| **Nom de l'option** | **Granularité** | **RTO** | **RPO** | **Documentation et tutoriels**|
+| --- | --- | --- | --- | --- |
+| - FTP backup ou Backup Storage est un espace de stockage de 500Go mis à disposition du client suite à la souscription au service.<br> - Le service doit être activé par le client.<br> - Aucune routine de backup n'est configurée par OVHcloud. | Au choix du client | Dépend du choix du client | N/A | [Utiliser Backup Storage sur un serveur dédié](https://docs.ovh.com/fr/dedicated/services-backup-storage/) |
+| - Backup Storage est une option de stockage supplémentaire qui permet d'avoir un espace disque supplémentaire pouvant atteindre 10To pour déposer des sauvegardes. | Au choix du client | Dépend du choix du client |  N/A | [Utiliser Backup Storage sur un serveur dédié](https://docs.ovh.com/fr/dedicated/services-backup-storage/) |
+
+## 5 - Logs
+
+> [!primary]
+> Consultez le guide [Premiers pas avec les API OVHcloud](https://docs.ovh.com/fr/api/first-steps-with-ovh-api/) pour vous familiariser avec l'utilisation des APIv6 OVHcloud.
+
+| **Source** | **Contenu** | **Liens** |
+| --- | --- | --- |
+| Control Plane (Espace Client) | Logs sur toutes les interactions réalisées via des appels API, lancés par les contacts administrateur, technique ou de facturation, sur les services auxquels ils ont accès. |- <https://api.ovh.com/console/#/me> (voir les appels `/me/api/logs`)<br>- [List of API calls done with your account](https://api.ovh.com/console/#/me/api/logs/self~GET)<br>- [List of API calls done on services you have access to](https://api.ovh.com/console/#/me/api/logs/services~GET) |
+| Service | Liste des tâches lancées sur un serveur donné | [Dedicated Servers ToDos](https://api.ovh.com/console/#/dedicated/server/%7BserviceName%7D/task~GET)|
+| Service | Liste des interventions lancées sur un serveur donné | [Technical interventions history](https://api.ovh.com/console/#/dedicated/server/%7BserviceName%7D/intervention~GET) |
+
+Les tâches sont des actions lancées par le client sur un Serveur Dédié : installation de l'OS, redémarrage de l'OS, activation du mode 'rescue', etc ...
+
+Les interventions sont des actions réalisées par les équipes OVHcloud dans les Datacentres sur les serveurs physiques : vérification de l'état de l'équipement, changement de CPU, RAM ou disque défectueux, etc ...
+
+## 6 - API
+
+| **Nom** | **Capacités** | **Liens** |
+| --- | --- | --- |
+| Control Plane et service | Manipulation des comptes client et des services sur lesquels le compte a des droits de gestion du service | https://api.ovh.com/console/#/dedicated/seed/servverconsole/#/dedicr |
```

Comment:
Hello @H4516022Y , this link is wrong. I could not find the correct one to replace it. Please check

Reply:
on my side it works but it could be replaced by this : https://api.ovh.com/console/#/dedicated/server
it conduct to the same page

Reply:
The original link you added (https://api.ovh.com/console/#/dedicated/seed/servverconsole/#/dedicr) does not redirect to any API call in particular. Anyway, I'll replace it with your suggesiton.